test/node/fast-deep-equal-tests.js

```diff
@@ -0,0 +1,389 @@
+// this is from fast-deep-equal spec/test.js
```

Comment:
for context, I had Samuel move all of the fast-deep-equal tests to a separate file so it was easier to keep up-to-date